docs/source/conf.py

```diff
@@ -132,7 +132,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
What is up here? Shall we add a comment, why static config is commented out?

Reply:
This was placed here but wasn't being used at all. This is just required if we want to overwrite sphinx's `default.css` file. This is already documented by the comment before. I just commented it back because the build was displaying a warning message because the `_static` directory does not exist.